src/box/lua/slab.cc

```diff
@@ -189,7 +187,7 @@ lbox_slab_info(struct lua_State *L)
 	 * quota_used_ratio > 0.9 work as an indicator
 	 * for reaching Tarantool memory limit.
 	 */
-	size_t arena_size = memtx->arena.used;
+
```

Comment:
The comment above doesn't make any sense now. Let's move it to `box_slab_info()`.

```diff
diff --git a/src/box/box.cc b/src/box/box.cc
index 8fa8272cfca0..44b283cc1644 100644
--- a/src/box/box.cc
+++ b/src/box/box.cc
@@ -4254,8 +4254,17 @@ box_slab_info(enum box_slab_info_type type)
 	case BOX_SLAB_INFO_ITEMS_USED:
 		return stats.small.used + stats.sys.used;
 	case BOX_SLAB_INFO_ARENA_SIZE:
+		/*
+		 * We could use stats.small.used + index_stats.total.used
+		 * here, but this would not account for slabs which are
+		 * sitting in slab cache or in the arena, available for reuse.
+		 * Make sure a simple formula: items_used_ratio > 0.9 &&
+		 * arena_used_ratio > 0.9 && quota_used_ratio > 0.9 work as
+		 * an indicator for reaching Tarantool memory limit.
+		 */
 		return memtx->arena.used;
 	case BOX_SLAB_INFO_ARENA_USED:
+		/** System allocator does not use arena. */
 		return stats.small.used + index_stats.totals.used;
 	case BOX_SLAB_INFO_QUOTA_SIZE:
 		return quota_total(&memtx->quota);
diff --git a/src/box/lua/slab.cc b/src/box/lua/slab.cc
index 2b62813be162..7b38eb41d58a 100644
--- a/src/box/lua/slab.cc
+++ b/src/box/lua/slab.cc
@@ -178,16 +178,6 @@ lbox_slab_info(struct lua_State *L)
 	/** How much address space has been already touched
 	 * (tuples and indexes) */
 	lua_pushstring(L, "arena_size");
-	/*
-	 * We could use totals.total + index_stats.total here,
-	 * but this would not account for slabs which are sitting
-	 * in slab cache or in the arena, available for reuse.
-	 * Make sure a simple formula:
-	 * items_used_ratio > 0.9 && arena_used_ratio > 0.9 &&
-	 * quota_used_ratio > 0.9 work as an indicator
-	 * for reaching Tarantool memory limit.
-	 */
-
 	luaL_pushuint64(L, arena_size);
 	lua_settable(L, -3);
 	/**
@@ -195,7 +185,6 @@ lbox_slab_info(struct lua_State *L)
 	 * data (tuples and indexes).
 	 */
 	lua_pushstring(L, "arena_used");
-	/** System allocator does not use arena. */
 	luaL_pushuint64(L, arena_used);
 	lua_settable(L, -3);
 
```

Reply:
Fix it